containers/Dockerfile.dnsmasq

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+FROM alpine:3.19.1
+
+RUN apk add --update --no-cache dnsmasq=2.90-r2 python3=3.11.8-r0 py3-pip=23.3.1-r0 py3-jinja2=3.1.2-r3
+
+COPY common/helpers.sh /helpers.sh
+COPY dnsmasq/entry-point.sh /entry-point.sh
+RUN chmod +x /entry-point.sh
+COPY dnsmasq/dnsmasq.conf.j2 /etc/dnsmasq.conf.j2
+# let our entry point write out the script
+RUN ln -sf /etc/dnsmasq.d/dnsmasq.conf /etc/dnsmasq.conf
+
+EXPOSE 53/udp
+EXPOSE 67/udp
```

Comment:
Probably should drop the port 53 since this PR makes it listen on 1053.

Reply:
And we listen on port 1067 for DHCP

Reply:
fixed in the rebase